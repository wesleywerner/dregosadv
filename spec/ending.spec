superbrief
go faster
gonear waterf
purloin golden egg
go south
go south
go south
