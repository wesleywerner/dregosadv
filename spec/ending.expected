  THIS GAME IS IN DEBUG MODE
  
  Dr Ego and the Egg of Man-Toomba
  An Interactive Fiction Story
  by Special Agent.
  Release 1 / Serial number 400410 / Inform v6.33 Library v6.12.3pre SX
  
  The Maro River (in the canoe)
. 
  In the canoe you can also see a trap (which is closed and empty), a fishing
  spear and a fishing net.
  
> >  Dr Ego and the Egg of Man-Toomba is now in its "superbrief" mode, which
  always gives short descriptions of locations (even if you haven't been
  there before).
  
> >  You hasten your journey.
  
  "Here we are", your guide says -- the canoe comes to a stop on the shore.
  
  River Shore (in the canoe)
  
  A jungle path is due north.
  
  In the canoe you can also see a trap (which is closed and empty), a fishing
  spear and a fishing net.
  
> >  Waterfall
  
  A temple is due south.
  A rope bridge is due west.
> >  [Purloined.]
  
> >  Temple
  
> >  Jungle Path
  
  A temple is due north.
  A clearing in the jungle is due west.
  The river shore is due south.
> >  River Shore
  
  A jungle path is due north.
  Your guide sits in the canoe gazing out over the moving river, the salty
  water gently lapping the sides of his craft.
  Your guide's eyes light up as you show the Golden Egg of Man-Toomba to him,
  you get in the canoe and spend the journey telling him of your adventure.
  
  You spend the next week cataloguing the precious artifact, before you
  donate the Golden Egg of Man-Toomba to the archeology museum.
  
      *** You have won ***
. 
  In that game you scored 0 out of a possible 108, in 4 turns.
  
  Would you like to RESTART, RESTORE a saved game, UNDO your last move, give
  the FULL score for that game, see some suggestions for AMUSING things to do
  or QUIT?
> >